cosmos/operators/local.py

```diff
@@ -321,6 +318,8 @@ def _configure_remote_target_path() -> tuple[Path | ObjectStoragePath, str] | tu
             return None, None
 
         if not settings.AIRFLOW_IO_AVAILABLE:
+            from airflow.version import version as airflow_version
+
             raise CosmosValueError(
                 f"You're trying to specify remote target path {target_path_str}, but the required "
                 f"Object Storage feature is unavailable in Airflow version {airflow_version}. Please upgrade to "
```

Comment:
```suggestion
raise CosmosValueError(
f"You're trying to specify remote target path {target_path_str}, but the required "
f"Object Storage feature is unavailable in Airflow version {AIRFLOW_VERSION}. Please upgrade to "
```

Would like to check if we can do this here?